assets/eip-5050/ERC5050Sender.sol

```diff
@@ -0,0 +1,180 @@
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.0;
+
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+import {Strings} from "@openzeppelin/contracts/utils/Strings.sol";
```

Comment:
```suggestion
```
EIP reference implementations shouldn't depend on mutable third party resources.  I suspect you can just drop these dependencies though and likely simplify the reference implementation.